app/build.gradle

```diff
@@ -83,7 +83,8 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
     // Markdown
-    implementation 'com.github.budsmile:bypass-android:1.0.5' //Needed for Ice Cream Sandwich (14&15)
+    implementation 'com.github.budsmile:bypass-android:1.0.5'
+    //Needed for Ice Cream Sandwich (14&15)
```

Comment:
we should revert formatting here. It can lead to someone believing the the comment is for `ru.noties:markwon:1.0.4` when it's actually for `com.github.budsmile:bypass-android:1.0.5`.

Reply:
I missed this, thanks, it is reverted now.